src/estimators.jl

```diff
@@ -319,6 +319,9 @@ mutable struct LGBMClassification <: LGBMEstimator
     predict_leaf_index::Bool
     predict_contrib::Bool
     predict_disable_shape_check::Bool
+    pred_early_stop::Bool
```

Comment:
As per documentation prediction early stopping only applies to classification and ranking applications.